apps/server/src/api-data/excel/excel.service.ts

```diff
@@ -43,9 +43,10 @@ export function generateRundownPreview(options: ImportMap): { rundown: OntimeRun
 
   const arrayOfData: unknown[][] = xlsx.utils.sheet_to_json(data, { header: 1, blankrows: false, raw: false });
 
-  const dataFromExcel = parseExcel(arrayOfData, getCustomFields(), options);
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
suggestion: we could avoid these changes by just having the `parseRundown` function ignore the `rundownMetadata`